src/main/webapp/stylesheets/feedStyle.css

```diff
@@ -1,88 +1,13 @@
-@import url('https://fonts.googleapis.com/css?family=Merriweather:400,400i,700,700i,900,900i');
-
-body {
-  font-family: 'Merriweather', serif;
-  font-size: 0.5em;
-  margin: 0px;
-  padding: 0px;
-}
-
-.wrap {
```

Comment:
were all of these already unused? seems like a lot of deleting but the code changes don't show you migrating away from it.

Reply:
Since the imported w3 css sheets have their own auto-styling I removed my old styling and replaced it with the smaller changes we wanted to make.